src/garage/torch/algos/loss_function_utils.py

```diff
@@ -1,10 +1,25 @@
-"""loss function utilities."""
+"""Differentiable utils typically used when computing loss functions."""
 import torch
 import torch.nn.functional as F
 
 
 def compute_advantages(discount, gae_lambda, max_len, baselines, rewards):
-    """Calculate advantages."""
+    """
+    Calculate advantages.
+
+    Advantages are a discounted cumulative sum.
+
+    Args:
```

Comment:
Please document comphrensivoely the expected shapes for baselines and rewards. It's very easy to provide this function with the wrong shape input tensor and get very surprising (and wrong) results.